code/game/objects/items/weapons/weaponry.dm

```diff
@@ -11,11 +11,58 @@
 	throwforce = 10
 	w_class = 2
 
+/obj/item/weapon/nullrod/nullstaff
+	name = "null staff"
+	desc = "A staff of pure obsidian, its very presence disrupts and dampens the powers of paranormal phenomenae."
+	icon_state = "nullstaff"
+	item_state = "nullstaff"
+	slot_flags = SLOT_BACK
+	w_class = 4
+
+/obj/item/weapon/nullrod/nullorb
+	name = "null sphere"
+	desc = "An orb of pure obsidian, its very presence disrupts and dampens the powers of paranormal phenomenae."
+	icon_state = "nullorb"
+	item_state = "nullorb"
+
+/obj/item/weapon/nullrod/nullathame
+	name = "null athame"
+	desc = "An athame of pure obsidian, its very presence disrupts and dampens the powers of paranormal phenomenae."
+	icon_state = "nullathame"
+	item_state = "nullathame"
+
+/obj/item/weapon/nullrod/itembox
+	name = "null item box"
+	desc = "A box to safe keep your religious items. What item did you bring to work today?"
+	icon = 'icons/obj/storage.dmi'
+	icon_state = "box"
+	item_state = "box"
+
+/obj/item/weapon/nullrod/itembox/attack_self(mob/user as mob)
+	if(..()) return
+
+	var/selection = input("Pick a null item type.") in list ("Rod","Staff", /*"Orb",*/ "Athame")
+	switch(selection)
+		if ("Rod")
+			new /obj/item/weapon/nullrod(usr.loc)

```

Comment:
Don't use `usr` in procs that are not directly exposed to the user. In this case, `user.loc` is better.